src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -68,6 +69,8 @@ const ChartsHolder = (props: {
   county: County;
   chartId: string;
   countyId: string;
+  selectedCounty: County;
+  setSelectedCounty: (input: string) => {};
```

Comment:
We need to pass these properties through the `ChartHolder` component so they are available for the `LocationMap` component, maybe we should move things outside the `ChartHolder` component?